pkg/kn/commands/domain/domain.go

```diff
@@ -0,0 +1,36 @@
+// Copyright © 2021 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package domain
+
+import (
+	"github.com/spf13/cobra"
+
+	"knative.dev/client/pkg/kn/commands"
+)
+
+// NewDomainCommand to manage domain mappings
+func NewDomainCommand(p *commands.KnParams) *cobra.Command {
+	domainCmd := &cobra.Command{
+		Use:     "domain COMMAND",
+		Short:   "Manage domain mappings",
+		Aliases: []string{"domains"},
+	}
+	domainCmd.AddCommand(NewDomainMappingCreateCommand(p))
+	domainCmd.AddCommand(NewDomainMappingDescribe(p))
```

Comment:
Why is this `NewDomainMappingDescribe` not `NewDomainMappingDescribeCommand` as with the others?

Reply:
Done in https://github.com/knative/client/pull/1267/commits/928e2cfb657dbb8011019125942a2270fc8614e7.